build/build.sh

```diff
@@ -207,6 +207,14 @@ edge_dir=${EDGE_DIR:-Edge}
 # Disable with '-disable-werror' option.
 cmake_flags+=" -DXRT_ENABLE_WERROR=$werror"
 
+# Add XRT version information
+# XRT_VERSION_RELEASE XRT_VERSION_MAJOR XRT_VERSION_MINOR are read from xrt_version.txt file
+# XRT_VERSION_PATCH is read from environment variable from cmake file
+
+XRT_VERSION_FILE=$BUILDDIR/xrt_version.txt
+source $XRT_VERSION_FILE
+cmake_flags+=" -Dxrt_version_release=${XRT_VERSION_RELEASE} -Dxrt_version_major=${XRT_VERSION_MAJOR} -Dxrt_version_minor=${XRT_VERSION_MINOR}"
```

Comment:
windows build need to use similar approach.  how does build_edge.sh gets the XRT version? 
It is better to define these variables in cmake itself so that, things will work even if someone calls cmake directly instead of build.sh.

Reply:
Hi @chvamshi-xilinx, made changes as suggested please review.